### ISSUE TITLE
Bug: Patch 4 causes "GetCameraObject not found!" errors

### ISSUE THREAD

**Issue by Laegel** [ersh1/BG3_NativeCameraTweaks#12]
Hey there, thanks for this plugin!

After BG3 Patch 4 installation, launching the game caused an error related to this plugin. 
Here is the log file content:
```
[11/02/23 16:38:18][info](Logger.hpp:198) Logger init - BG3NativeCameraTweaks 20202
[11/02/23 16:38:18][info](Settings.cpp:99) Config loaded
[11/02/23 16:38:18][critical](Hooks.h:25) GetCameraObject not found!
```
Note: I am on Linux. I do not think this is related; still worth mentioning.

**Comment by lYokal** [comment]
Same for me (Windows) 

[11/03/23 00:48:44][info](Logger.hpp:198) Logger init - BG3NativeCameraTweaks 20202
[11/03/23 00:48:44][info](Settings.cpp:99) Config loaded
[11/03/23 00:48:44][critical](Hooks.h:25) GetCameraObject not found!

**Comment by ersh1** [comment]
Fixed in 2.2.3
